credit_card.go

```diff
@@ -39,6 +39,7 @@ type CreditCard struct {
 	UniqueNumberIdentifier    string             `xml:"unique-number-identifier,omitempty"`
 	BillingAddress            *Address           `xml:"billing-address,omitempty"`
 	Subscriptions             *Subscriptions     `xml:"subscriptions,omitempty"`
+	ProductId                 string             `xml:"product_id,omitempty"`
```

Comment:
`ProductID` to stick with Go's naming conventions.
`product-id` for the XML to keep the serialization consistent. Both works, but we've been trying to use `-` consistently.